Chinese/little_prince/this_is_a_story_about_a_boy.md

```diff
@@ -0,0 +1 @@
+The Little Prince is a play based on the book of the same name by Antoine de Saint-Exupéry, adapted by Rick Cummins and John Scoullar before 2000. Rick Cummins wrote the music, and John Scoullar wrote the script and lyrics. There are several changes from the book, including the omission of the drunkard (tippler), switchman and merchant characters; the removal of a great deal of the narration from the aviator; significant changes to the rose scenes; and a large change in the order of events.
```

Comment:
It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.